web/themes/custom/cloudy/pattern-lab/css/pattern-scaffolding.css

```diff
@@ -96,7 +96,7 @@ details {
 .pl-card-deck {
```

Comment:
Where again are we using this? I see `cloudy-tile-deck` in our examples. Maybe this is something we could safely remove (the pl prefix makes me think its some legacy pattern lab styles we aren't using)

Reply:
Good catch. We removed .pl-card-deck from the pattern lab scss.